Resources/Prototypes/_NF/Shipyard/pathfinder.yml

```diff
@@ -0,0 +1,27 @@
+- type: vessel
+  id: Pathfinder
+  name: NT Pathfinder
+  description: Once a scout ship serving with the Nanotrasen Marine Expeditionary Forces, this now decommissioned expedition capable ship can be yours!
+  price: 48920
```

Comment:
if we could get it above 50k for an expedition vessel. Dont even have to add anything just make it over 50k just to get to a planet, the ship is pretty bare bones enough to make the other options worthwhile